backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -28,12 +28,33 @@ public async Task<Guid> CreateProject(CreateProjectInput input)
                 RetentionPolicy = input.RetentionPolicy,
                 Users = input.ProjectManagerId.HasValue ? [new() { UserId = input.ProjectManagerId.Value, Role = ProjectRole.Manager }] : [],
             });
+        // Also delete draft project, if any
+        await dbContext.DraftProjects.Where(dp => dp.Id == projectId).ExecuteDeleteAsync();
```

Comment:
Because if the admin never looks at the draft project page, and just clicks the link in the email he's been sent, we still want to delete the draft project. So putting it here means we don't repeat ourselves (PromoteDraftProject calls this), and the draft project is deleted in the same transaction that creates the real project (so there's never a chance of half-created projects with a draft still hanging around). I ensured that the draft project had the same GUID as the "real" project that would eventually be created from it, to ensure that this very simple logic was all we'd need.

Reply:
PromoteDraftProject mutation has been removed in commit 1ee9bc0b0122fc698baba08328b7bfb88c22f8d2, so this is now the only place to put the draft project removal.